sklearn/svm/classes.py

```diff
@@ -553,6 +553,22 @@ class SVC(BaseSVC):
     intercept_ : array, shape = [n_class * (n_class-1) / 2]
         Constants in decision function.
 
+    fit_status_ : int
+        0 if correctly fitted,
+        1 otherwise (will raise warning)
+
+     probA_, probB_ : array, shape = [n_class * (n_class-1) / 2]
+
+     If probability=True, the parameters learned in Platt scaling to
```

Comment:
The indentation seems off. Also you can refer to the user guide that also explains that, though it's good to also mention the paper.